tools/operator/testdata/barebones-test-input32.yml

```diff
@@ -53,7 +53,7 @@ invokes:
       timeout: 240000
     ccOpt:
       ccType: ccchecker
-      keyStart: _KEYSTART_
+      keyStart: 0
```

Comment:
Need to add 
`      keyIdx: [1]
      keyPayload: [2]`
after keyStart option.

Reply:
ok